manifests/0000_70_cluster-network-operator_03_deployment.yaml

```diff
@@ -47,6 +47,10 @@ spec:
           value: "quay.io/openshift/origin-kuryr-cni:4.3"
         - name: KURYR_CONTROLLER_IMAGE
           value: "quay.io/openshift/origin-kuryr-controller:4.3"
+        - name: NETWORK_METRICS_DAEMON_IMAGE
+          value: "quay.io/openshift/origin-network-metrics-daemon:4.5"
+        - name: KUBE_RBAC_PROXY_IMAGE
+          value: "quay.io/openshift/origin-kube-rbac-proxy:4.4"
```

Comment:
`KUBE_RBAC_PROXY_IMAGE` is already earlier in the file

Reply:
Will remove it.